<ui version="4.0" >
 <class>mdlgEMvars</class>
 <widget class="QDialog" name="mdlgEMvars" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Variables</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>250</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="mlblTitle" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Select Variables for Model Fitting:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>70</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QCheckBox" name="mchkBoxSp" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Sp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxXC" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>XCorr</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mchkBoxDCn" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>DelCn2</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="mrBtn2DEM" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>2 Dimensional Model</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="mrBtn3DEM" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>210</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>3 Dimensional Model</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mdlgEMvars</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdlgEMvars</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
